releases/release-1.28/release-notes/maps/pr-117055-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 117055
+releasenote:
+  text: The feature gates `CSIMigrationGCE` is graduated to GA and were unconditionally
+    enabled have been removed in v1.25, and the entire `gcepd` package has been removed.
```

Comment:
```suggestion
  text: The feature gate `CSIMigrationGCE` graduated to GA and has been unconditionally enabled since `v125`, and the entire `gcepd` package has been removed.
```